src/libp2p/chain_exchange/message.rs

```diff
@@ -258,15 +280,127 @@ fn fts_from_bundle_parts(
 
 #[cfg(test)]
 mod tests {
+    use super::*;
+    use crate::blocks::{Block, BlockHeader};
+    use crate::db::MemoryDB;
+    use crate::libp2p::chain_exchange::{CompactedMessages, TipsetBundle};
+    use crate::message::SignedMessage;
+    use crate::shim::{
+        address::Address,
+        crypto::Signature,
+        message::{Message, Message_v3},
+    };
+    use num::BigInt;
     use quickcheck_macros::quickcheck;
     use serde_json;
 
-    use super::*;
-
     #[quickcheck]
     fn chain_exchange_response_status_roundtrip(status: ChainExchangeResponseStatus) {
         let serialized = serde_json::to_string(&status).unwrap();
         let parsed = serde_json::from_str(&serialized).unwrap();
         assert_eq!(status, parsed);
     }
+
+    #[test]
+    fn tipset_bundle_save_load_roundtrip() {
```

Comment:
What are you testing here?

Reply:
Removed with the no-chain-exchain-forwarding change